azure-kusto-data/azure/kusto/data/_token_providers.py

```diff
@@ -180,9 +177,7 @@ async def _get_token_async():
 
             if token is None:
                 async with self._async_lock:
-                    token = await KustoTracing.call_func_tracing_async(
-                        self._get_token_impl_async, name_of_span=f"{self.name()}.get_token_impl_async", tracing_attributes=context
-                    )
+                    token = await Span.run_async(self._get_token_impl_async, f"{self.name()}.get_token_impl_async", context)

```

Comment:
I think here and in other palces where it's long it's good to break it up to new lines

Reply:
addressed